vllm/v1/core/sched/scheduler.py

```diff
@@ -304,8 +304,8 @@ def schedule(self) -> SchedulerOutput:
                     break
 
             if new_blocks is None:
-                # Cannot schedule this request.
-                break
+                # Continue scheduling subsequent requests
+                continue
```

Comment:
Should we only try to schedule subsequent requests when priority scheduling is enabled?
@WoosukKwon